css/components/service-card.css

```diff
@@ -0,0 +1,31 @@
+.service-card{
+    display: inline-block;
+    width: 300px;
+    height: 320px;
```

Comment:
Korteles aukstis bus pagal jo turinio dydi, o plotis pagal jam skirta grid stulpelio vieta``

Reply:
Šitą suprantu. Bet jie dinamiški ir mąsčiau kad reikės pririšti prie bendro tos sekcijos dizaino, kai jis bus daromas